students/johnpharmd/lesson07/html_render.py

```diff
@@ -0,0 +1,50 @@
+class Element(object):
+    """renders an html element"""
+
+    tag = ''
+    indent = '    '
+
+    def __init__(self, content=''):
+        self.open_tag = '<' + self.tag + '>'
+        self.close_tag = '</' + self.tag + '>'
```

Comment:
since you're not going to use `self.open_tag` and `self.close_tag` in various methods there is not reason to put it in `__init__`, instead have it as local vars in `render` method